components/nimbus/src/enrollment.rs

```diff
@@ -2198,6 +2283,47 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    fn test_multi_feature_per_branch_conflict() -> Result<()> {
+        let mut test_experiments = get_test_experiments();
+        // conflicting_experiments conflict with themselves, but not with
+        // test_experiments
+        let mut conflicting_experiments = get_conflicting_multi_feature_experiments();
+        test_experiments.append(&mut conflicting_experiments);
```

Comment:
I think this is likely a smell where we're composing `Vec`s of `Experiment`s in the tests. Unless there's a good reason, I'd prefer not to be doing this.